.github/workflows/ci-automated-pipeline.yaml

```diff
@@ -0,0 +1,155 @@
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  auto-pr-triage:
+    runs-on: ubuntu-latest
+    name: Triage for automated PRs
+    if: |
+      contains(github.event.pull_request.title, 'Release') &&
+      github.actor == 'bitnami-bot'
+    steps:
+      # Enables auto-merge and adds necessary labels for automated releases' PRs
+      - id: labeling
+        name: Label PR
+        uses: andymckay/labeler@1.0.4
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          add-labels: "verify, auto-merge"
+      - id: auto-merge
+        name: Enable auto-merge
+        run: |
+          curl --request POST \
+          --url https://api.github.com/graphql \
+          --header 'authorization: Bearer ${{ secrets.BITNAMI_BOT_TOKEN }}' \
+          --data '{
+            "query": "mutation { enablePullRequestAutoMerge(input: {pullRequestId: \"${{ github.event.pull_request.node_id }}\", mergeMethod: SQUASH}) { clientMutationId }}"
+            }' \
+          --fail
+  get-container:
+    runs-on: ubuntu-latest
+    name: Get modified containers
+    needs: auto-pr-triage
+    outputs:
+      container: ${{ steps.get-container.outputs.container }}
+      result: ${{ steps.get-container.outputs.result }}
+      path: ${{ steps.get-container.outputs.path }}
+    steps:
+      - id: get-container
+        name: Get modified containers
+        run: |
+          # Using the Github API to detect the files changed as git merge-base stops working when the branch is behind
+          # and jitterbit/get-changed-files does not support pull_request_target
+          URL="https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files"
+          files_changed_data=$(curl -s --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -X GET -G "$URL")
+          files_changed="$(echo $files_changed_data | jq -r '.[] | .filename')"
+          # Adding || true to avoid "Process exited with code 1" errors
+          containers_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "^bitnami/[^/]*" | sort | uniq || true)"
+          flavors=($(echo "$files_changed" | xargs dirname | grep -o "^bitnami/[^/]*/[^/]*/[^/]*" | sort | uniq || true))
+          container_name=$(echo "$containers_dirs_changed" | sed "s|bitnami/||g")
+
+          if [[ "${#flavors[@]}" -eq "1" ]]; then
+            # Changes done in only one container -> OK
+            echo "::set-output name=result::ok"
+            echo "::set-output name=container::${container_name}"
+            echo "::set-output name=path::${flavors[0]}"
+          else
+            # No changes detected or there are changes done in more than one container -> SKIP
+            echo -e "::set-output name=error::No changes detected or there are changes done in more than one container:\n${containers_dirs_changed}\n. The rest of the tests will be skipped."
+            echo "::set-output name=result::skip"
```

Comment:
I'm not sure we need this validation for automated PRs; they are expected to only modify 1 flavor.

Having said that, we could always leave this validation to check for a faulty PR created by our internal tools, but in that case, we need to modify the `auto-pr-review` logic. As it is now, setting the `result=skip` will make all the following jobs skip, including the `auto-pr-review` and no reviewer would be assigned to the PR.

Reply:
You right!, I'll revisit this

Reply:
Updated, if there are more than one "flavor" changed the PR will fail and manual intervention will be required